static/css/jira-configuration.css

```diff
@@ -234,6 +234,15 @@
   color: #455570 !important;
 }
 
+.restart-backfill-button {
+   /* override aui styling */
+  padding: 0 !important;
```

Comment:
I usually try and avoid using `!important`. Is there another way to do this?

Reply:
There are ways to target elements more specifically and override items through this. Can do a pair programming session to discuss this.

Reply:
I usually try to avoid using `!important` as well (I actually have a blog post about the use of it being something you should avoid). Unfortunately, we are using Atlassian's AUI components and the default styling comes from those. Using `!important` is the only way to override these styles here, hence the comment `/* override aui styling */`

Reply:
Updated and created a ticket to remove the other `!important` entries from css. Used those initially when setting up the new UI because the AUI components were giving me a headache.  That should probably be cleaned up now though.

Reply:
Awesome, thank you :)